interfaces/supabase.py

```diff
@@ -29,19 +31,33 @@ def update(self, update, query_key, query_value):
     def insert(self, data):
         data = self.client.table(self.table).insert(data).execute()
```

Comment:
Possible case of `self.table` being equal to `''`

Reply:
addressed in supabase refactor, class no longer specifies table on initialisation